tests/mka/test_mka_stats.py

```diff
@@ -0,0 +1,126 @@
+import pytest
```

Comment:
Somehow snappi_ixnetwork group mail I have missed as that is way down within Group. Anyway, best way to check this huge feature to test all field. I will check these changes. 
At the same time in slack I have mentioned - snappi_ixnetwork properly compacted MacSec in topology level.
Create multiple similar OTG MacSec config within a single port. All IxNetwork MacSec  DG should compacted to single DG with proper multiplier and MV.